src/depackers/bunzip2.c

```diff
@@ -129,7 +117,7 @@ static unsigned int get_bits(struct bunzip_data *bd, char bits_wanted)
     // If we need to read more data from file into byte buffer, do so
     if (bd->inbufPos == bd->inbufCount) {
       if (0 >= (bd->inbufCount = hio_read(bd->inbuf, 1, IOBUF_SIZE, bd->in_fd))) /* libxmp: read -> hio_read */
-        error_exit("input EOF");
+        longjmp(bd->jmpbuf, RETVAL_UNEXPECTED_INPUT_EOF); /* libxmp: error_exit -> longjmp */
```

Comment:
So, this is the only place we need longjmp: Instead of that, can we not
move the returned bits to parameters and return 0/-1 for success/failure?

Reply:
Sigh.. I told that, but it looks like pain to do that. Ignore...